tests/api/v1/samples/put.js

```diff
@@ -39,25 +42,29 @@ describe(`tests/api/v1/samples/put.js, PUT ${path} >`, () => {
     u.doCustomSetup('COFFEE', 'BAGEL')
     .then((samp) => Sample.create(samp))
     .then((samp) => {
-      sampleName = samp.name;
+      sampleName1 = samp.name;
+      subjectId1 = samp.subjectId;
+      aspectId1 = samp.aspectId;
       return u.doCustomSetup('TEA', 'TOFFEE');
     })
     .then((samp) => Sample.create(samp))
     .then((samp) => {
-      subjectId = samp.subjectId;
-      aspectId = samp.aspectId;
+      sampleName2 = samp.name;
```

Comment:
You are not using sampleName2 anywhere else other than L25 and L52.

Reply:
I added it for consistency. I needed to access both aspect1 and aspect2, and it's clearer to call it sampleName1 and sampleName2 than to just have one sampleName.